processing/src/test/java/org/apache/druid/math/expr/ConstantExprTest.java

```diff
@@ -34,5 +35,6 @@ public void testLongSingleThreadedExpr()
     assertNotSame(expr.eval(null), expr.eval(null));
     Expr singleExpr = Expr.singleThreaded(expr);
     assertSame(singleExpr.eval(null), singleExpr.eval(null));
+    assertEquals(expr.toString(), singleExpr.toString());
```

Comment:
We should have a test for `stringify` too as that method is expected to be a legit API. Would also be nice to have tests for other types of constant expr, and tests that the `stringify` ends up as specific strings (rather than just that the orig and single-threaded ones match).

Reply:
extended the test to cover all of them!